src-electron/upgrade/upgrade.js

```diff
@@ -39,15 +46,53 @@ const fsp = fs.promises
 async function getForcedExternalStorage(db, attributeId) {
   let pkgs = await queryPackage.getPackageRefByAttributeId(db, attributeId)
   let zcl = await queryPackage.getPackageByPackageId(db, pkgs)
-  zcl = zcl.path
-  let obj = await fsp.readFile(zcl)
-  let data = JSON.parse(obj)
+  zcl = zcl?.path
```

Comment:
Somewhere along the way this part is also eating up an "exception condition" (zcl being null or undefined).